### PR TITLE
LPS-122373 | LPS-105491 buildLang

### PR DESCRIPTION
**Pull request by gabrielpramos**
7.4 only.

### CODE REVIEW

File: modules/apps/app-builder/app-builder-web/src/main/resources/META-INF/resources/js/pages/apps/edit/ListItems.es.js [hunk 1]
```diff
@@ -32,21 +33,40 @@ const ListItems = ({defaultLanguageId, itemType, items}) => {
 	const {
 		dispatch,
 		state: {
-			app: {dataLayoutId, dataListViewId},
+			app: {dataLayoutId, dataListViewId, workflowProcessId},
 		},
 	} = useContext(EditAppContext);
 
-	const itemId = itemType === 'DATA_LAYOUT' ? dataLayoutId : dataListViewId;
+	const itemsProps = {
```
Comment (albinojunior) [inline modules/apps/app-builder/app-builder-web/src/main/resources/META-INF/resources/js/pages/apps/edit/ListItems.es.js:40]: @gabrielpramos, What do you think about moving the specific logic for out of this component to make it more generic and putting this properties on the component API?

**Review by albinojunior — CHANGES_REQUESTED** [review]

**Comment by gabrielpramos** [comment]
ci:test:sf

**Comment by gabrielpramos** [comment]
ci:test:relevant

**Comment by gabrielpramos** [comment]
ci:test:app-builder

**Comment by gabrielpramos** [comment]
cc @interaminense @kevenleone

File: modules/apps/app-builder/app-builder-web/src/main/resources/META-INF/resources/js/pages/apps/edit/ListItems.es.js (hunk 1 above)
Comment (gabrielpramos) [reply modules/apps/app-builder/app-builder-web/src/main/resources/META-INF/resources/js/pages/apps/edit/ListItems.es.js]: done 👍

File: modules/apps/app-builder/app-builder-web/src/main/resources/META-INF/resources/js/pages/apps/edit/ListItems.es.js [hunk 2]
```diff
@@ -78,12 +57,14 @@ const ListItems = ({defaultLanguageId, itemType, items}) => {
...
-						<Cell>{fromNow(dateModified)}</Cell>
+						<Cell>{dateCreated && fromNow(dateCreated)}</Cell>
+						<Cell>{dateModified && fromNow(dateModified)}</Cell>
 						<Cell align={'right'}>
 							<ClayRadioGroup
 								inline
-								onSelectedValueChange={() => onItemIdChange(id)}
+								onSelectedValueChange={(id) =>
+									onItemIdChange(id)
+								}
```
Comment (albinojunior) [inline modules/apps/app-builder/app-builder-web/src/main/resources/META-INF/resources/js/pages/apps/edit/ListItems.es.js:67]: Hey @gabrielpramos , what do you think about just call the `onChange` function? 

Like this:
`onSelectValueChange={onChange}`

File: modules/apps/app-builder/app-builder-web/src/main/resources/META-INF/resources/js/pages/apps/edit/EditAppBody.es.js [hunk 3]
```diff
@@ -14,15 +14,33 @@
...
 import ListItems from './ListItems.es';
 
-export default ({defaultLanguageId, endpoint, title, ...restProps}) => {
+export default ({
+	defaultLanguageId,
+	endpoint,
+	itemType,
+	title,
+	...restProps
+}) => {
```
Comment (albinojunior) [inline modules/apps/app-builder/app-builder-web/src/main/resources/META-INF/resources/js/pages/apps/edit/EditAppBody.es.js:34]: @gabrielpramos, what do you think about export it as a named component? And rename it to `EditAppBodyStep` ?

Like this:

`export default function EditAppBodyStep({...}) `

**Review by albinojunior — CHANGES_REQUESTED** [review]

**Comment by gabrielpramos** [comment]
ci:stop

**Pull request closed without merging**